### DISCUSSION TITLE
nerdctl cp command in MacOS always fail to find GNU tar

### DISCUSSION THREAD

**Discussion by GodDrinkTeJAVA** [rancher-sandbox/rancher-desktop#3291] [Q&A] [answered]
I'm using MacOS on Apple Silicon and nerdctl cp command always tells me that /bin/tar as GNU tar is not found. The message is written as below.
```WARN[0000] "/bin/tar" does not seem GNU tar```
The problem is, it seems to be in /usr/bin/tar.
Would this be the problem with nerdctl or the virtual machine? If it is the problem from virtual machine, could you tell me if there is any way to fix this?

**Comment by adamkpickering** [comment, marked as answer]
`nerdctl` runs inside the VM, which only has busybox tar at the moment. We should be able to install GNU tar in the VM pretty easily, which should fix this issue. I'm going to create an issue from this so we can track this work. Thanks for finding this!

**Comment by jandubois** [comment]
As a workaround you can install GNU tar manually:

```console
$ rdctl shell sudo apk add tar
fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.16/community/x86_64/APKINDEX.tar.gz
(1/2) Installing libacl (2.3.1-r0)
(2/2) Installing tar (1.34-r0)
Executing busybox-1.35.0-r17.trigger
OK: 311 MiB in 95 packages
```

Unfortunately this will be wiped out every time the VM is booted (because the `/usr` directory is on a RAM disk), so you have to re-run the command after each restart of Rancher Desktop.

**Comment by GodDrinkTeJAVA** [comment]
Thank you for the kind answers! I wish both of those would be marked as answer, but seems only one of them could 😢
 I would look forward to the new update! 😀
